modules/outbound-command-event-handler/src/domain/bulk_transaction_agg/handlers/process_sdk_outbound_bulk_party_info_request.ts

```diff
@@ -26,14 +26,15 @@
 
 import { ILogger } from '@mojaloop/logging-bc-public-types-lib';
 import {
+    BulkTransactionInternalState,
     CommandEvent,
+    IndividualTransferInternalState,
     ProcessSDKOutboundBulkPartyInfoRequestCmdEvt,
     PartyInfoRequestedDmEvt,
 } from '@mojaloop/sdk-scheme-adapter-private-shared-lib';
-import { BulkTransactionAgg } from '..';
+import { BulkTransactionAgg } from '@module-domain';
```

Comment:
Does this work fine?

Don't we have the self-reference issue that we had to resolve the other day?

Reply:
Good catch. Yes it does seem to work fine but I've changed it here and throughout just incase.